app/controllers/users_controller.rb

```diff
@@ -5,5 +5,6 @@ def index
 
   def show
     @user = User.find(params[:id])
+    @recent_posts = @user.recent_posts
```

Comment:
- [ ] In your user model, the method you used to get recent posts was `latest_posts`. Kindly change this to `@user.latest_posts` to avoid errors.